docs/form-datasets.rst

```diff
@@ -146,3 +147,23 @@ For internal datasets, the instance name is the ``list_name`` specified on the *
   Due to a pyxform limitation, it is necessary for there to be some value in the `choice_filter` column (for at least one question) when referencing internal datasets. If none of the questions in your form need filtering, put `true()` as the `choice_filter` value.
 
 For external datasets, the instance name is the filename specified in the ``select_one_from_file`` or ``select_multiple_from_file`` declaration without the file extension. For example, to look up a ward's label given the form :ref:`above <selects-from-csv>`, the instance name to use is ``wards`` because the filename referenced is ``wards.csv``. The expression would be ``instance("wards")/root/item[name = ${ward}]/label``. 
+
+.. _form-datasets-attaching-csv:
+
+Attaching CSVs for lookups without selection
```

Comment:
I don't love the use of "selection" here and in the first sentence.

"Attaching CSVs for lookups without a select" and "directly without using a select" read better to me.